scripts/email-config.js

```diff
@@ -30,7 +31,9 @@ const KEYS = {
 const VALID_SERVICES = new Set([ 'sendgrid', 'ses', 'socketlabs' ])
 const VALID_PROPERTIES = new Map([
   [ 'percentage', value => value >= 0 && value <= 100 ],
-  [ 'regex', value => value && typeof value === 'string' ]
+  [ 'regex', value =>
+    value && typeof value === 'string' && value.indexOf('"') === -1 && safeRegex(value)
```

Comment:
You talked about a terminating quote, this rejects if any quote exists?

Reply:
Yep. It's terminating because the regex is evaluated as a string literal, so any quote present would open an injection vulnarability.

Reply:
I mean:

https://github.com/mozilla/fxa-auth-server/blob/c21f6fdd52350bef81f3c6c6cb2cd2507d0cd7ce/lib/senders/email.js#L514